release/authorization/README.md

```diff
@@ -0,0 +1,420 @@
+# gNMI Authorization Protocol
+
+## Objective
+
+This proto definition and the reference code(to be delivered seperately) serve
+to describe an authorization framework for controlling which gNMI paths of a
+network device users can access. The authorization policy is initially intended
+to be deployed to a device, with the ability to define:
+
+*   Policy rules - each rule defines a single authorization policy.
+*   Groups of users - as a method to logically group users in the administrative
+    domain, for instance: operators or administrators.
+*   Users which are individual users as part of either rules or in groups.
```

Comment:
nit: rewrite for consistency as "Users - individuals referenced in rules or group definitions"

Reply:
done